Resources/public/css/theme/views/testone.css

```diff
@@ -0,0 +1,23 @@
+.ez-testoneview-article + .ez-testoneview-article {
+    border-top: 1px solid #eee;
+}
+
+.ez-testoneview-article {
+    padding: 10px 5px;
+}
+
+.ez-testoneview-article.ez-testoneview-active {
+    background-color: lightgray;
+}
+
+.ez-view-testonearticlesview {
+    padding: 10px;
+}
+
+.ez-view-testoneintroview h4 {
+    font-size: 1.4em;
+}
+
+.ez-view-testoneintroview {
+    padding: 10px;
```

Comment:
To samo co w: https://github.com/gbalcewicz/PlatformUIBundle/pull/1/files#r41733821